src/commonMain/kotlin/com/benasher44/uuid/uuid.kt

```diff
@@ -1,7 +1,29 @@
+@file:Suppress("RedundantVisibilityModifier")
+
 package com.benasher44.uuid
 
-internal val UUID_BYTES = 16
-internal val UUID_STRING_LENGTH = 36
+internal const val UUID_BYTES = 16
+internal const val UUID_STRING_LENGTH = 36
+
+/**
+ * Construct new [UUID] instance using the given data.
+ *
+ * @param msb The 64 most significant bits of the [UUID].
+ * @param lsb The 64 least significant bits of the [UUID].
+ */
+// @SinceKotlin("1.x")
+@Suppress("FunctionName")
+public fun UUID(msb: Long, lsb: Long): UUID =
```

Comment:
Should this match the updated capitalization?

Reply:
Yeah, as I wrote above. Will update tomorrow.

Reply:
Ah sorry! Missed that :)